scripts/start

```diff
@@ -4,6 +4,11 @@ if [ -z "$(docker network ls --filter name=^ffc-pay$ --format={{.Name}})" ]; the
   docker network create ffc-pay
 fi
 
+if [ -z "$(docker container ls --filter name=ffc-pay-statement-azurite  --format={{.ID}})" ]; then
+  echo "Azurite container already exists, skipping creation"
+  args="--scale ffc-pay-statement-azurite=0"
+fi
```

Comment:
This is the main part of this PR.  If there is already a container running named `ffc-pay-statement-storage` then don't create a conflicting container with the same name when starting this repo.  ie, scale the number of instances to 0.

The result of this will mean that this repo will just use the existing container rather than creating a conflicting one.  Therefore both services will be able to access the same files for local development.

Making the same changes in the publisher repo also means it doesn't matter which way around these services start, they will always be able to access storage.   Also allows them to be worked on independently.